### PR TITLE
21 ACR cards

### PR DESCRIPTION
**Pull request by Fulgur14**

### CODE REVIEW

File: hemlock_vial.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Name:Hemlock Vial
+ManaCost:1 B
+Types:Artifact
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigDraw | TriggerDescription$ When CARDNAME enters the battlefield, you draw a card and you lose 1 life.
+SVar:TrigDraw:DB$ Draw | Defined$ You | NumCards$ 1 | SubAbility$ DBLoseLife
+SVar:DBLoseLife:DB$ LoseLife | LifeAmount$ 1
+A:AB$ PumpAll | Cost$ B T Sac<1/CARDNAME> | ValidCards$ Creaturee.YouCtrl+equipped,Equipment.YouCtrl | KW$ Deathtouch | SpellDescription$ Each equipped creature and Equipment you control gains deathtouch until end of turn.
```
Comment (tool4ever) [inline hemlock_vial.txt:7]: valid typo

Comment (Fulgur14) [reply hemlock_vial.txt]: Fixed.

File: the_aesir_escape_valhalla.txt [hunk 2]
```diff
@@ -0,0 +1,9 @@
+Name:The Aesir Escape Valhalla
+ManaCost:2 G
+Types:Enchantment Saga
+K:Chapter:3:I,II,III
+SVar:I:DB$ ChangeZone | ChangeNum$ 1 | ChangeType$ Permanent.YouOwn | Mandatory$ True | Origin$ Graveyard | Destination$ Exile | SubAbility$ DBGainLife | SpellDescription$ Exile a permanent card from your graveyard. You gain life equal to its mana value.
+SVar:DBGainLife:DB$ GainLife | Defined$ You | LifeAmount$ ExiledWith$CardManaCost
+SVar:II:DB$ PutCounter | ValidTgts$ Creature.YouCtrl | TgtPrompt$ Select target creature you control | CounterType$ P1P1 | CounterNum$ ExiledWith$CardManaCost | SpellDescription$ Put a number of +1/+1 counters on target creature you control equal to the mana value of the exiled card.
+SVar:III:DB$ ChangeZone | ChangeType$ Card.StrictlySelf,ExiledWith | Origin$ Battlefield,Exile | Destination$ Hand | SpellDescription$ Return The Aesir Escape Valhalla and the exiled card to their owner's hand.
```
Comment (tool4ever) [inline the_aesir_escape_valhalla.txt:8]: was that ChangeType tested?

Comment (tool4ever) [inline the_aesir_escape_valhalla.txt:8]: CARDNAME

Comment (Fulgur14) [reply the_aesir_escape_valhalla.txt:8]: No, I just asked on Discord if it would work.

File: yggdrasil_rebirth_engine.txt [hunk 3]
```diff
@@ -0,0 +1,11 @@
+Name:Yggdrasil, Rebirth Engine
+ManaCost:3
+Types:Legendary Artifact
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigChangeZoneAll | TriggerDescription$ When CARDNAME enters the battlefield, exile all creature cards from your graveyard.
+SVar:TrigChangeZoneAll:DB$ ChangeZoneAll | ChangeType$ Card.YouOwn | Origin$ Graveyard | Destination$ Exile | SubAbility$ DBChooseCard
+A:AB$ Dig | Cost$ T | Defined$ You | DigNum$ 3 | ChangeNum$ All | DestinationZone$ Exile | SpellDescription$ Exile the top three cards of your library.
+A:AB$ ChooseCard | Cost$ 4 T | Choices$ Creature.ExiledWithSource | ChoiceZone$ Exile | SorcerySpeed$ True | SubAbility$ DBChangeZone | SpellDescription$ Put a creature card exiled with Yggdrasil onto the battlefield under your control. It gains haste until end of turn. Activate only as a sorcery.
```
Comment (tool4ever) [inline yggdrasil_rebirth_engine.txt:7]: NICKNAME

File: yggdrasil_rebirth_engine.txt [hunk 4]
```diff
@@ -0,0 +1,11 @@
+Name:Yggdrasil, Rebirth Engine
+ManaCost:3
+Types:Legendary Artifact
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigChangeZoneAll | TriggerDescription$ When CARDNAME enters the battlefield, exile all creature cards from your graveyard.
+SVar:TrigChangeZoneAll:DB$ ChangeZoneAll | ChangeType$ Card.YouOwn | Origin$ Graveyard | Destination$ Exile | SubAbility$ DBChooseCard
```
Comment (tool4ever) [inline yggdrasil_rebirth_engine.txt:5]: bad sub

File: yggdrasil_rebirth_engine.txt (hunk 3 above)
Comment (tool4ever) [inline yggdrasil_rebirth_engine.txt:7]: don't see why extra ChooseCard effect is needed

File: the_aesir_escape_valhalla.txt (hunk 2 above)
Comment (tool4ever) [reply the_aesir_escape_valhalla.txt]: `Card.ExiledWithSource`

File: yggdrasil_rebirth_engine.txt (hunk 3 above)
Comment (tool4ever) [reply yggdrasil_rebirth_engine.txt]: why resolve, still needs change?

Comment (Fulgur14) [reply yggdrasil_rebirth_engine.txt]: Hm, you mean that it could be just ChangeZone without ChooseCard? I might have understood it wrongly.

Comment (tool4ever) [reply yggdrasil_rebirth_engine.txt]: yea I meant it like that

File: yggdrasil_rebirth_engine.txt [hunk 5]
```diff
@@ -0,0 +1,11 @@
+Name:Yggdrasil, Rebirth Engine
+ManaCost:3
+Types:Legendary Artifact
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigChangeZoneAll | TriggerDescription$ When CARDNAME enters the battlefield, exile all creature cards from your graveyard.
+SVar:TrigChangeZoneAll:DB$ ChangeZoneAll | ChangeType$ Creature.YouOwn | Origin$ Graveyard | Destination$ Exile
+A:AB$ Dig | Cost$ T | Defined$ You | DigNum$ 3 | ChangeNum$ All | DestinationZone$ Exile | SpellDescription$ Exile the top three cards of your library.
+A:AB$ ChooseCard | Cost$ 4 T | Choices$ Creature.ExiledWithSource | ChoiceZone$ Exile | SorcerySpeed$ True | SubAbility$ DBChangeZone | SpellDescription$ Put a creature card exiled with NICKNAME onto the battlefield under your control. It gains haste until end of turn. Activate only as a sorcery.
+SVar:DBChangeZone:DB$ ChangeZone | Defined$ ChosenCard | Origin$ Exile | Destination$ Battlefield | GainControl$ True | AnimateSubAbility$ DBPump | SubAbility$ DBCleanMinus
```
Comment (tool4ever) [inline yggdrasil_rebirth_engine.txt:8]: AnimateSubAbility is wrong

File: yggdrasil_rebirth_engine.txt [hunk 6]
```diff
@@ -0,0 +1,10 @@
+Name:Yggdrasil, Rebirth Engine
+ManaCost:3
+Types:Legendary Artifact
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigChangeZoneAll | TriggerDescription$ When CARDNAME enters the battlefield, exile all creature cards from your graveyard.
+SVar:TrigChangeZoneAll:DB$ ChangeZoneAll | ChangeType$ Creature.YouOwn | Origin$ Graveyard | Destination$ Exile
+A:AB$ Dig | Cost$ T | Defined$ You | DigNum$ 3 | ChangeNum$ All | DestinationZone$ Exile | SpellDescription$ Exile the top three cards of your library.
+A:AB$ ChangeZone | Cost$ 4 T | Origin$ Exile | Destination$ Battlefield | Mandatory$ True | ChangeType$ Card.ExiledWithSource | SorcerySpeed$ True | GainControl$ True | SubAbility$ DBPump | RememberChanged$ True | SpellDescription$ Put a creature card exiled with Yggdrasil onto the battlefield under your control. It gains haste until end of turn. Activate only as a sorcery.
```
Comment (tool4ever) [inline yggdrasil_rebirth_engine.txt:7]: now NICKNAME again :p

File: arno_dorian.txt
Comment (tool4ever) [inline arno_dorian.txt:1]: looks like none of these are getting merged in the right directory? :/

**Pull request closed without merging**